source/administration/object-management/object-versioning.rst

```diff
@@ -30,6 +30,13 @@ new version of that object with a unique version ID. MinIO marks the "latest"
 version of the object that clients retrieve by default. Clients can then
 explicitly choose to list, retrieve, or remove a specific object version. 
 
+.. versionchanged:: 2023-08-04T17-40-21Z
+
+   MinIO restricts object versioning to no more than 10,000 versions of each object.
+
+If an operation would create more than 10,000 versions of the object, the operation results in an error.
```

Comment:
This might imply bulk ops.

```suggestion
If a write operation would exceed the 10,000 object version limit, MinIO blocks the operation and returns an error.
```

If an operation would create more than 10,000 versions of the object, the operation results in an error.
```